coderd/apikey.go

```diff
@@ -216,9 +216,20 @@ func (api *API) tokens(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	var apiKeys []codersdk.APIKey
+	var apiKeys []codersdk.ConvertedAPIKey
 	for _, key := range keys {
-		apiKeys = append(apiKeys, convertAPIKey(key))
+		user, err := api.Database.GetUserByID(ctx, key.UserID)
```

Comment:
We should use `GetUsersByIDs` here instead, otherwise, you could execute hundreds of separate queries if there are hundreds of users.

Generally, we should never query in `for` loops!